easybuild/tools/utilities.py

```diff
@@ -56,7 +56,7 @@ def flatten(lst):
     return res
 
 
-def quote_str(x):
+def quote_str(x, esc_newline=False):
```

Comment:
if you want to keep this short and non-descriptive, go with `esc_nl`, otherwise use `escape_newline` (not a mix of both)